index.bs

```diff
@@ -11596,7 +11596,7 @@ example, the sound could be omnidirectional, in which case it would
 be heard anywhere regardless of its orientation, or it can be more
 directional and heard only if it is facing the listener.
 {{AudioListener}} objects (representing a person's
-ears) have <em>forward</em> and <em>up</em> vectors
+ears) have [=front|forward=] and [=up=] vectors
```

Comment:
Perhaps `front` should be `forward` as well?

Reply:
Yeah, I'll do that. I was just preserving the existing text.  But I'll have to check other places that refer to "front" and change them to forward.  Maybe I'll do that in a different PR?

Reply:
I think it's better to leave this as is in the PR.  If we really want to, a follow-up PR can change all references from "front" to "forward".

Reply:
SGTM